app/src/androidTest/java/fr/free/nrw/commons/AboutActivityTest.kt

```diff
@@ -63,11 +60,8 @@ class AboutActivityTest {
 
     @Test
     fun testLaunchRateUs() {
-        val appPackageName = InstrumentationRegistry.getInstrumentation().targetContext.packageName
         Espresso.onView(ViewMatchers.withId(R.id.about_rate_us)).perform(ViewActions.click())
         Intents.intended(IntentMatchers.hasAction(Intent.ACTION_VIEW))
-        Intents.intended(CoreMatchers.anyOf(IntentMatchers.toPackage(Urls.PLAY_STORE_PACKAGE_NAME),
-                IntentMatchers.hasData("${Urls.PLAY_STORE_URL_PREFIX}$appPackageName")))
```

Comment:
hasData was still a valuable test

Reply:
@macgills This actually fails the test if the app is installed on the device because then the browser will not launch.

Reply:
We want to be asserting `hasData` whether it be for the playstore or for the browser, you can use `anyOf(hasData(playStore), hasData(browser))`